scripts/plots/cam/exevs_refs_grid2obs_cape_plots.sh

```diff
@@ -268,7 +269,7 @@ chmod +x run_all_poe.sh
 # Run the POE script in parallel or in sequence order to generate png files
 #**************************************************************************
 if [ $run_mpi = yes ] ; then
-   mpiexec -np 840 -ppn 84 --cpu-bind verbose,depth cfp ${DATA}/run_all_poe.sh
+   mpiexec -np 768 -ppn 86 --cpu-bind verbose,depth cfp ${DATA}/run_all_poe.sh
```

Comment:
The refs_grid2obs_cape jobs are requesting `select=9:ncpus=86`, so 9 * 86 is 774